packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -122,13 +122,30 @@ export type VisibilityState = (typeof VisibilityState)[keyof typeof VisibilitySt
  * @sealed
  */
 export interface IContainerRuntimeBaseEvents extends IEvent {
-	(event: "batchBegin", listener: (op: ISequencedDocumentMessage) => void);
-	/**
-	 * @param runtimeMessage - tells if op is runtime op. If it is, it was unpacked, i.e. it's type and content
+	(
```

Comment:
Nit: Can we document the `batchBegin` and `batchEnd` events while we're here?

Reply:
Sorry, not right now, I'm about to go on leave for a month, trying to wrap up!  Will leave a to-do to revisit when making the breaking change.